definitions/npm/sha3_v1.2.x/flow_v0.56.x-/sha3_v1.2.x.js

```diff
@@ -0,0 +1,11 @@
+declare module "sha3" {
+  declare class SHA3Hash {
+    constructor(hashLength?: 224 | 256 | 384 | 512): SHA3Hash;
+    update(data: any, inputEncoding?: 'binary' | 'ascii' | 'utf8'): void;
+    digest(type?: 'hex' | 'binary'): string;
+  }
+
+  declare module .exports: {
```

Comment:
Remove the space in `module .exports`

Reply:
fixed, I blame setting the save hotkey to 'format and save' in jetbrains >.>